package.json

```
@@ -34,7 +34,7 @@
     "typescript": "^4.9.4"
   },
   "scripts": {
-    "compile-wasm": "turbo run compile",
+    "compile-wasm": "pnpm --filter @anonklub/merkle-tree compile",
```

I liked it better before.
Why do you want to skip turbo?

`turbo run compile` wouldn't indeed work if called directly in your terminal if you installed it only locally. In a pnpm project, to run the binaries normally in `node_modules/.bin` you need to execute them in a pnpm scripts or use `pnpm exec`.

turbo will pick the scripts called `[name]` in the underlying packages. And then you can wrap `turbo run [name]` in an `pnpm` script.

This executes fine the `@anonklub/spartan-ecdsa` and `@anonklub/merkle-tree` for me
 
```suggestion
    "compile": "turbo run compile"
```

You need to adjust the turbo config.json so that the cache works properly though

---

I see, I agree that this looks better but the only drawback is that you will need to have turbo installed globally. And that is not a good practice, its okay for short-term but for long-term we need a better way I think. 
That's why thought of separating the Rust wasm compilation from the turbo stuff so that I can easily do `pnpm install`.

---

> you will need to have turbo installed globally

You don't. I don't have turbo installed globally and this script is working fine for me